### DISCUSSION TITLE
在海外VPS上挂载阿里云盘握手超时

### DISCUSSION THREAD

**Discussion by MegaBunLi** [AlistGo/alist#4202] [Q&A]
我是在Oracle的VPS上用alist挂阿里网盘，一开始能挂上去，但是过一段时间就打不开了，显示握手超时，这种情况该怎么解决

**Comment by MegaBunLi** [comment]
状态: 
Post "https://open.aliyundrive.com/adrive/v1.0/user/getDriveInfo": net/http: TLS handshake timeout

**Reply by Alashen2023** [reply]
解决了没？我也遇到这种情况，在硬路由docker下，网络也没科学，只是升级了下路由器版本

**Comment by anwen-anyi** [comment]
应该无解，阿里从上线那一刻就是禁止海外IP访问/使用的
（或者你换IP试试看只能，应该是这样）

**Comment by wweggplant** [comment]
我想问一下,这个视频播放的话,还走vps流量吗? 如果直连的话, 是不是像海外的vps只是影响浏览页面的速度? 播放视频就是直连网盘了?

**Reply by Simonzhang8** [reply]
302模式就不走vps流量

**Comment by JJTools** [comment]
阿里云不限制挂海外，我一个挂在香港一个挂在美国。如何IP被拉黑了。尝试用CF 套一层出口试试

**Comment by tank-long** [comment]
我的115超时，夸克就还好
